smartsim/settings/arguments/launch/pals.py

```diff
@@ -158,5 +158,5 @@ def set(self, key: str, value: str | None) -> None:
             )
             return
         if key in self._launch_args and key != self._launch_args[key]:
```

Comment:
And the last one I think!
```suggestion
        if key in self._launch_args and value != self._launch_args[key]:
```
?